draft-ietf-webtrans-http2.md

```diff
@@ -308,7 +300,9 @@ shortest possible encoding. For example, for the frame types defined in this
 document, this means a single-byte encoding, even though it is possible to
 encode these values as a two-, four-, or eight-byte variable-length integer.
 
-## WT_PADDING Frames
+## WT_PADDING Frames {#WT_PADDING}
+
+*[WT_PADDING]: #WT_PADDING
```

Comment:
Super cool! Just curious: why is this one `#WT_PADDING` while the rest are `#` only?

Reply:
They mean the same. But I forgot to fix this one up.

Reply:
(Cleaned up in [d73be38](https://github.com/ietf-wg-webtrans/draft-webtransport-http2/pull/35/commits/d73be38b1ac9272109b068a9b417d6e89c7cc05f))